Dockerfile

```diff
@@ -23,22 +23,26 @@ RUN dnf module -y install python39 && dnf install -y python39 python39-pip
 
 RUN mkdir /plugin
 RUN chmod 777 /plugin
-ADD smallfile_plugin.py /plugin
-ADD smallfile_schema.py /plugin
-ADD test_smallfile_plugin.py /plugin
-ADD requirements.txt /plugin
-ADD smallfile-example.yaml /plugin
-ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugins/main/LICENSE /plugin
+ADD README.md /plugin/
+ADD poetry.lock /plugin/
+ADD pyproject.toml /plugin/
+ADD smallfile_plugin.py /plugin/
+ADD smallfile_schema.py /plugin/
+ADD test_smallfile_plugin.py /plugin/
+ADD smallfile-example.yaml /plugin/
+ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugins/main/LICENSE /plugin/
```

Comment:
Please point this either to the license file in this repo or in the new plugin python template repo. The arcaflow-plugins repo will be retired.

Reply:
While testing adding the local LICENSE file I ran into errors with formatting as the carpenter check specifically looks for `"(ADD|COPY) .*/LICENSE /.*":` . This means that to add the local file I would explicitly need to `ADD ./LICENSE`  instead of the regular convention of `ADD LICENSE`. We will need to discuss how we want to handle the license and update accordingly for the carpenter check. This will prevent users from breaking their builds and having to open the image-builder project to find the problem.

Reply:
updated